Sources/Tabs/HomeCoordinator.swift

```diff
@@ -29,4 +33,24 @@ final class HomeCoordinator: NSObject,
         navController.setViewControllers([developerMenuVC], animated: true)
         root.present(navController, animated: true)
     }
+    
+    private func createCellModels() -> [[TabbedViewCellModel]] {
+        #if DEBUG
+        let developerModel = TabbedViewCellModel(cellTitle: GDSLocalisedString(stringLiteral: "Developer Menu")) {
```

Comment:
Should be just able to pass the string as `GDSLocalizedString` conforms to `ExpressibleByStringLiteral`